tools/editor/editor_settings.cpp

```diff
@@ -587,7 +587,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	set("3d_editor/zoom_modifier",4);
 	hints["3d_editor/zoom_modifier"]=PropertyInfo(Variant::INT,"3d_editor/zoom_modifier",PROPERTY_HINT_ENUM,"None,Shift,Alt,Meta,Ctrl");
 	set("3d_editor/emulate_numpad",false);
-	set("3d_editor/trackpad_hint", false);
+	set("3d_editor/enable_trackpad", false);
```

Comment:
That could work. I was also thinking of "Emulate 3-Button Mouse" like in Blender. What do you think of that?